profiles/lacuna_stories/lacuna_stories.info

```diff
@@ -3,7 +3,7 @@ name = "Lacuna Stories"
 description = "Installation profile for Lacuna Stories."
```

Comment:
Oops, does that text show up during the install process?  I was intentionally trying to only change forward-facing text because the profile is still called lacuna_stories and I wanted to keep that internal stuff consistent, but I can change everything in the profile to be just 'lacuna' if you want?